app/controllers/admin_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
 
   def newsletter
     authorize! :manage, :all
-    @members = Member.confirmed.wants_newsletter.all
+    @members = Member.confirmed.undeleted.wants_newsletter.all
```

Comment:
Could we use a default scope instead? All queries become undeleted members, instead of changing all queries.